python/python-core/control-flow-i/indentation-and-commenting.md

```diff
@@ -9,11 +9,13 @@ links:
 practiceQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
```

Comment:
they can answer `this is true` and that would be incorrect because we're checking for quotes as well. in general this isn't a good question for type
```suggestion
```